18/bullseye/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM buildpack-deps:bullseye
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+ENV NODE_VERSION 18.0.0
+
+RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+  && case "${dpkgArch##*-}" in \
```

Comment:
Is there a particular reason why the indentation in the full debian Docker files differs from that in the slim editions?

Reply:
Not particularly, it's just coming from the template file